elliott/elliottlib/cli/create_cli.py

```diff
@@ -133,25 +136,42 @@ async def create_cli(
     if sum(map(bool, [date, batch_id])) != 1:
         raise click.BadParameter("Need either --date or --batch-id")
 
-    if "boilerplates" not in et_data:
-        raise ValueError("`boilerplates` is required in erratatool.yml")
+    common_advisory_template = yaml.safe_load(
+        requests.get(
+            "https://raw.githubusercontent.com/openshift-eng/ocp-build-data/refs/heads/docs-update/release_notes.yml"
+        )
+    )
+
+    common_boilerplate = common_advisory_template.get("boilerplates", {}).get(art_advisory_key)
+    group_boilerplate = et_data.get("boilerplates", {}).get(art_advisory_key)
```

Comment:
Think we need an override, but not sure if `group` is the right granularity. Think we run the risk of keeping that in group.yml. Is it possible/desirable to have only a assembly based override to not get tempted for durable fixes in the wrong place?

Reply:
By group, I was referring to the erratatool.yml file found in the "group" level, i.e. OCP version: https://github.com/openshift-eng/ocp-build-data/blob/openshift-4.19/erratatool.yml